website/content/docs/enterprise/pkcs11-provider/oracle-tde.mdx

```diff
@@ -114,13 +114,7 @@ TDE is complex, but an example way to enable it is:
 
 1. Open a `sqlplus` session into the root container (or switch into it with `ALTER SESSION SET CONTAINER = CDB$ROOT;`).
 
-1. Set WALLET_ROOT and TDE_CONFIGURATION parameters on the Oracle database.
-
-<Note>
-  The wallet root directory is only used to set the TDE configuration parameter.
-  To learn more about the wallet parameters refer to the
-  <a href="https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/TDE_CONFIGURATION.html">Oracle TDE documentation</a>.
-</Note>
+1. Set WALLET_ROOT and TDE_CONFIGURATION parameters on the Oracle database. The wallet root directory is only used to set the TDE configuration parameter. To learn more about the wallet parameters refer to the <a href="https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/TDE_CONFIGURATION.html"> Oracle TDE documentation</a>.
```

Comment:
```suggestion
1. Set WALLET_ROOT and TDE_CONFIGURATION parameters on the Oracle database. The wallet root directory is only used to set the TDE configuration parameter. To learn more about the wallet parameters refer to the [Oracle TDE documentation](https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/TDE_CONFIGURATION.html).
```

The HTML notation was required because of a bug when working with the new aside notation. If we're just presenting it as markdown, we should use markdown link notation.